monitoring/grafana/dashboards/dashboards.yml

```
@@ -0,0 +1,12 @@
+apiVersion: 1
+
+providers:
+  - name: 'default'
+    orgId: 1
+    folder: ''
+    type: file
+    disableDeletion: false
+    updateIntervalSeconds: 10
+    allowUiUpdates: true
+    options:
+      path: /etc/grafana/provisioning/dashboards 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)